README.md

```diff
@@ -63,7 +79,7 @@ This will provide a json config block to add to your `.juttle/config.{js,json}`
 This will also use the gmail nodejs api to read the list of labels
 assocated with the authenticated user, to verify that the token was created successfully.
 
-## Add the appropriate configuration items to `.juttle/config.{js,json}`
+### Add the appropriate configuration items to `.juttle/config.{js,json}`
```

Comment:
Consider choosing one extension `json` and using that throughout. If folks want to get creative I believe this is documented elsewhere.